### PR TITLE
metro field in metal_reserved_ip_block

### PR DESCRIPTION
**Pull request by t0mk**
This PR adds `metro` param to reserved IP block. It also bumps packngo dep to 0.10.0.

#28 
#55

### CODE REVIEW

**Comment by t0mk** [comment]
@displague 

I saw this is Github actions build log:
```
Error: ../../../go/pkg/mod/github.com/packethost/packngo@v0.10.0/packngo.go:257:22: resp.Header.Values undefined (type http.Header has no field or method Values)
note: module requires Go 1.15
```

Is that a problem?

**Comment by displague** [comment]
> Is that a problem?

I don't think it should be a problem to update the go build version. Let's find out in the GH action and update the versions we build with throughout.

https://github.com/hashicorp/terraform-provider-google-beta/blob/master/go.mod#L36

File: metal/resource_metal_reserved_ip_block.go [hunk 1]
```diff
@@ -72,9 +72,16 @@ func resourceMetalReservedIPBlock() *schema.Resource {
 		ForceNew: true,
 	}
 	reservedBlockSchema["facility"] = &schema.Schema{
-		Type:     schema.TypeString,
-		Optional: true,
-		ForceNew: true,
+		Type:          schema.TypeString,
+		Optional:      true,
+		ForceNew:      true,
+		ConflictsWith: []string{"metro"},
```
Comment (displague) [inline metal/resource_metal_reserved_ip_block.go:78]: Let's get descriptions added while we are here.

**Comment by t0mk** [comment]
@displague I fixed your review comments

**Pull request merged**